test/cli-test.ts

```diff
@@ -477,4 +477,8 @@ parallel('Spellchecker CLI', function testSpellcheckerCLI(this: { timeout(n: num
     const result = await runWithArguments('--config test/fixtures/config/basic.jsonc');
     result.should.not.have.property('code');
   });
+  it ('can read options from `package.json`', async () => {
+    const result = await runCommand("node build/index.js");
+    result.should.not.have.property('code');
+  })
```

Comment:
Nit:
```suggestion

  it ('can read options from `package.json`', async () => {
    const result = await runCommand("node build/index.js");
    result.should.not.have.property('code');
  })
```

Reply:
I have a suggestion for this tests. I'd like to keep the tests independent from the rest of the project - I don't think they should run on the project's documentation or using the project's `package.json`. I'd rather they run using test files in `test/fixtures`.

So what I'd suggest is adding a second argument to `runWithArguments`, a `string?` that can be used to override the value of the `APP_ROOT_PATH` environment variable in `runCommand`. 

Then we'd create a test `package.json` in `test/fixtures/config` that's independent of the project's `package.json`, and change the test to look like:

```suggestion

  it ('can read options from `package.json`', async () => {
    const result = await runWithArguments('', `${__dirname}/fixtures/config`);
    result.should.not.have.property('code');
  })
```

Reply:
> I have a suggestion for this tests. I'd like to keep the tests independent from the rest of the project - I don't think they should run on the project's documentation or using the project's package.json. I'd rather they run using test files in test/fixtures.
So what I'd suggest is adding a second argument to runWithArguments, a string? that can be used to override the value of the APP_ROOT_PATH environment variable in runCommand.


Completely agree, working on fix.